tap_oracle/__init__.py

```diff
@@ -552,8 +562,23 @@ def main_impl():
          filter_schemas = args.config.get('filter_schemas').split(',')
       do_discovery(conn_config, filter_schemas)
 
-   elif args.catalog:
+   elif args.properties:
       state = args.state
+
+      # Sort the properties
+      streams = args.properties['streams']
+      for stream in streams:
+            new_properties = {}
+            old_properties = stream['schema']['properties']
+            order = stream['column_order']
+
+            for column in order:
+               new_properties[column] = old_properties[column]
+
+            stream['schema']['properties'] = new_properties
+
+      args.catalog = Catalog.from_dict(args.properties)
+      
       do_sync(conn_config, args.catalog, args.config.get('default_replication_method'), state)
```

Comment:
>      do_sync(conn_config, args.catalog, args.config.get('default_replication_method'), state)
should this change to args.properties ? would the rest work fine ?

Reply:
`args.catalog = Catalog.from_dict(args.properties)`
It convert args.properties, which was read from the json file as a dictionary, to args.catalog, which is a Catalog instance needed by the rest.

Reply:
@tomwang-varicent thanks for clarifying.
So plz just update all the readme files for the taps you changed to have the new commands